src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -42,9 +43,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException {
     try {
       String url = request.getParameter(URL_PARAMETER);
-      // TODO: add numComments parameter. Comment count will be hard-coded until then.
+      String numCommentsInput = request.getParameter(NUM_COMMENTS_PARAMETER);
+      // If nothing is inputted, length of list is default, otherwise it's what was inputted.
+      long numComments =
```

Comment:
why do we use long instead of integer?

Reply:
That is what the API requires. So, for simplicity's sake, I keep everything a Long since it makes no difference.